### ISSUE TITLE
Running Voilà into docker on a Synology NAS (SSL/TLS websocket error)

### ISSUE THREAD

**Issue by yb23** [voila-dashboards/voila#1229]
Hi,

I have been running Voilà into a docker container since some weeks. It was serving the files on a port (8890) which was mapped to external port 8890 of the server, that I access through a reverse proxy. 
I access it using url : https://sub.domain.fr and all is working correctly.
But now I obtained a new domain name and I forwarded this new domain name to the port 9999 using reverse proxy. I mapped my container from internal 8890 to server 9999 and when I try to access Voilà using the url https://newdomain.com, it creates a websocket connection error and nothing is displayed.
To recap :
- access through https://ip-address-of-server:9999 works fine
- access through https://old-domain-name:9999 works fine
- access through https://new-domain-name doesn't work
Is there some way to change the urls used for websocket connections ?
Or had someone an idea from where it could come ?
Thanks a lot for your help.
